### PR TITLE
io_uring: add basic io_uring clickhouse perf test

### PR DESCRIPTION
**Pull request by tomershafir**
Add a basic isolated io_uring Clickhouse perf test, similar to `tests/performance/mmap_io.xml`, to get a rough understanding of the performance of Clickhouse on io_uring, aligned to the current testing strategy.

### Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/performance/io_uring.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<test>
+    <settings>
+        <local_filesystem_read_method>io_uring</local_filesystem_read_method>
+    </settings>
+
+    <create_query>CREATE TABLE hits_none (WatchID UInt64 CODEC(NONE)) ENGINE = MergeTree ORDER BY tuple()</create_query>
+    <fill_query>INSERT INTO hits_none SELECT WatchID FROM test.hits</fill_query>
+    <fill_query>OPTIMIZE TABLE hits_none FINAL</fill_query>
```
Comment (nickitat) [inline tests/performance/io_uring.xml:8]: what is the purpose of this? i.e. why not just read from `test.hits`?

Comment (nickitat) [reply tests/performance/io_uring.xml:8]: oh, sorry, I didn't noticed `CODEC(NONE)`. then the question is why we need codec `None`?)

Comment (tomershafir) [reply tests/performance/io_uring.xml]: probably a bad copy paste, as I dont see a real reason for it, fixed

**Review by nickitat — APPROVED** [review]

**Comment by tomershafir** [comment]
@nickitat can you `add can tested`?

**Pull request merged**